src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -851,6 +852,17 @@
     )
     private boolean helpMarkdown = false;
 
+    @Option(
+            name = "--instrument_code",
+            usage = "Enable code instrumentation to perform code coverage analysis. Options are: "
+            + " 1. NONE (deault)"
```

Comment:
Can you run compiler with --help and see how this is printed? This will be a single line, maybe you can you add "\n" at the end?